src/RacingGame.js

```diff
@@ -0,0 +1,41 @@
+import RacingCars from './models/RacingCars.js';
+import RaceStatusDisplay from './RaceStatusDisplay.js';
+import { GAME_SETTINGS } from './constants/Symbol.js';
+
+class RacingGame {
+  #racingCars;
+  #attempt;
+
+  constructor(carNameList, roundAttempt) {
+    this.#racingCars = new RacingCars(carNameList);
+    this.#attempt = roundAttempt;
+  }
+
+  startRace() {
+    RaceStatusDisplay.printRaceStartMessage();
+
+    for (
+      let Racing = GAME_SETTINGS.RACING_START_POINT;
+      Racing < this.#attempt;
+      Racing++
+    ) {
+      this.#racingCars.moveCarsInRound();
+      RaceStatusDisplay.printRoundStatus(this.#racingCars.getCars());
+    }
+  }
```

Comment:
물론 성능적으로는 기본 for 문이 훨씬 좋을 듯 합니다만, 
이 부분에 있어서 Racing 자체를 단순히 순회 목적으로만 사용하신다면, 기본 for 문 말고 Airbnb JS 스타일 가이드에서 지향하는 자바스크립트 고차 함수를 더 사용해보는 건 어떨까요?
단순히 제 의견이니까 참고만 해주세요!
[Airbnb JS 스타일 가이드 - 이터레이터와 제너레이터](https://github.com/airbnb/javascript?tab=readme-ov-file#iterators-and-generators)

```suggestion
    Array.from({ length: this.#attempt - GAME_SETTINGS.RACING_START_POINT }, (_, index) => {
  this.#racingCars.moveCarsInRound();
  RaceStatusDisplay.printRoundStatus(this.#racingCars.getCars());
});
```

Reply:
오.. 코드도 수정해주시고 너무 감사해요! 아직 고차함수에 익숙하지가 않다보니 for문으로 해야겠다 라고 생각했는데 이렇게 고차함수로 구현하는 방법도있네요! 해당 예시는 제가 잘 공부해보고 사용하도록 할게요!
이런 의견은 항상 환영입니다 감사해요!